Source/Core/Core/IOS/DolphinDevice.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <algorithm>
+#include <cstring>
+
+#include "Common/Logging/Log.h"
+#include "Common/Timer.h"
+#include "Common/scmrev.h"
+#include "Core/Core.h"
+#include "Core/HW/Memmap.h"
+#include "Core/IOS/DolphinDevice.h"
+
+namespace IOS::HLE::Device
+{
+IPCCommandResult GetSystemTime(const IOCtlVRequest& request);
```

Comment:
An anonymous namespace can be used here around the enum and the definition of these functions to allow the removal of these redundant prototypes.

Reply:
added anonymous namespace in bb0caef.